apps/clinical/src/services/__tests__/stopMedicationService.test.ts

```
@@ -1,19 +1,69 @@
-import { get, post } from '@bahmni/services';
+import {
+  get,
+  post,
+  createEncounterBundle,
```

ESLint

[error] 'createEncounterBundle' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] `../stopMedicationService` import should occur after import of `../../utils/fhir/encounterResourceCreator`

(import/order)